test/build.test.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('chai').assert;
+const fs = require('fs');
+
+describe('vl-util build', function() {
+
+  const assertDatDeInhoudVanDeDistFolderOvereenkomtMetDeExpectedFolder = (distFolder, expectedFolder) => {
+    fs.readdir(distFolder, function (err, files) {
```

Comment:
ik zou ook het aantal files in beide folders vergelijken, anders is er nog de mogelijkheid dat er te veel of te weinig files in de dist folder zitten.